Code/Framework/AzCore/AzCore/IO/Path/Path.inl

```diff
@@ -185,6 +197,10 @@ namespace AZ::IO::Internal
     template <typename InputIt, typename EndIt, typename = AZStd::enable_if_t<AZStd::Internal::is_input_iterator_v<InputIt>>>
     static constexpr bool IsAbsolute(InputIt first, EndIt last, const char preferredSeparator)
     {
+    #if defined(AZ_TRAIT_CUSTOM_PATH_ROOT_SEPARATOR)
+        const AZStd::string_view path{ first, last };
+        return path.find(AZ_TRAIT_CUSTOM_PATH_ROOT_SEPARATOR) != AZStd::string_view::npos;
+    #else
```

Comment:
Repeat: This check should be inside of the `if (preferredSeprator == '/')` on line 208.

Also can you do me a favor an update that check to be `if (preferredSeparator == PosixPathSeparator)`

Reply:
As above, happy to change it but figure it might one day be needed for a non-posix platform; just let me know what you prefer.

Will update to use PosixPathSeparator regardless.

Reply:
This was updated as well.